src/Utils/PluginInfo.as

```diff
@@ -121,4 +141,29 @@ class PluginInfo
 			return;
 		}
 	}
+
+	bool LoadChangelog() {
+		Net::HttpRequest@ req = API::Get("plugin/" + m_siteID + "/versions");
+		Json::Value js = Json::Parse(req.String());
```

Comment:
This function runs synchronously, and thus will block the UI and cause a small freeze. It should be requested asynchronously. You should copy what `PluginTab` is doing with its `m_request` handle, or consider doing this as a coroutine instead (but that's a bigger thing that I'd then also wanna do for `m_request`).

Reply:
yeah originally i was trying to do that, i had some skill issues tho and couldnt get it going... just tried again copypasting away and it works perfectly...fml